<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>610</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="densityPlot" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="velocityPlot" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="pressurePlot" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="energyPlot" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="testCombo">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>470</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Toro 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Toro 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Toro 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Toro 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Toro 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MiniProject A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MiniProject B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MiniProject C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MiniProject D</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="schemeCombo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>78</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FORCE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FLIC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SLIC</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="limiterCombo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>78</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Minbee</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vanleer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Superbee</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="rho0">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>530</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="u0">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>530</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-10000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="p0">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>530</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="gammaA">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.400000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rho1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>560</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.125000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="u1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>560</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="p1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>560</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="gammaB">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>470</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.400000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>510</y>
      <width>471</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> Interface pos    Density              Velocity              Pressure            Material</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Cells">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>570</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>570</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>540</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="CFL">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>540</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.900000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Time">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>510</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.005000000000000</double>
    </property>
    <property name="value">
     <double>0.250000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>510</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>T</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveResults">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Results</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>test.out</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isobaricFix">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Isobaric Fix</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="GFMCombo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Riemann Ghost Fluid Method</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Original Ghost Fluid Method</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="p3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>620</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rho2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>590</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="u2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>590</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rho3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>620</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.125000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="p2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>590</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="u3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>620</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QComboBox" name="mat0">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>530</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="mat1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>560</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="mat3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>620</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="mat2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>590</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="x0">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>570</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.550000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>600</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.600000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="Interfaces">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of interfaces</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>470</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma B</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="levelsetPlot" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEuler_equations_in_1D">
    <property name="title">
     <string>Euler equations in 1D</string>
    </property>
   </widget>
   <addaction name="menuEuler_equations_in_1D"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
